src/reconciliation.h

```diff
@@ -0,0 +1,335 @@
+#include <minisketch/include/minisketch.h>
+
+/** Static component of the salt used to compute short txids for transaction reconciliation. */
+static const std::string RECON_STATIC_SALT = "Tx Relay Salting";
+/** Used to convert a floating point reconciliation coefficient q to an int for transmission. Specified by BIP-330. */
+static constexpr uint16_t Q_PRECISION{(2 << 14) - 1};
+/**
+ * Interval between sending reconciliation request to the same peer.
+ * This value allows to reconcile ~100 transactions (7 tx/s * 16s) during normal system operation at capacity.
+ * More frequent reconciliations would cause significant constant bandwidth overhead due to
+ * reconciliation metadata (sketch sizes etc.), which would nullify the efficiency.
+ * Less frequent reconciliations would introduce high transaction relay latency.
+ */
+static constexpr std::chrono::microseconds RECON_REQUEST_INTERVAL{16s};
```

Comment:
nit: use smth like:

```
static constexpr std::chrono::microseconds RECON_REQUEST_INTERVAL{16000000};
```

using the `"" s` operator requires a `using namespace std::chrono_literals` or similar: [https://en.cppreference.com/w/cpp/chrono/operator%22%22s](https://en.cppreference.com/w/cpp/chrono/operator%22%22s).

Same for `RECON_RESPONSE_INTERVAL`.